script.js

```diff
@@ -6,26 +6,63 @@ function padNumber(number) {
   return number.toString().padStart(2, '0');
 }
 
+function generateShowHTML(show) {
+  return `
+    <article class="show-card" data-show-id="${show.id}">
+      <img src="${show.image?.medium || 'https://via.placeholder.com/210x295'}" alt="${show.name}">
+      <div class="show-content">
+        <h2>${show.name}</h2>
+        <p><strong>Genres:</strong> ${show.genres.join(', ')}</p>
+        <p><strong>Status:</strong> ${show.status}</p>
+        <p><strong>Rating:</strong> ${show.rating.average || 'N/A'}</p>
+        <p><strong>Runtime:</strong> ${show.runtime} minutes</p>
+        <p>${show.summary || 'No description available.'}</p>
+      </div>
+    </article>`;
+}
+
 function generateEpisodeHTML(episode) {
   const episodeNumber = `S${padNumber(episode.season)}E${padNumber(episode.number)}`;
   return `
-    <article class="episode-card">
-        <div class="episode-badge">${episodeNumber}</div>
-        <img src="${episode.image?.medium || 'https://via.placeholder.com/210x295'}" alt="${episode.name}">
-        <div class="episode-content">
-            <h2>${episode.name}</h2>
-            <p>${episode.summary || 'No description available.'}</p>
-        </div>
+    <article class="episode-card" data-episode-id="${episode.id}">
+      <div class="episode-badge">${episodeNumber}</div>
+      <img src="${episode.image?.medium || 'https://via.placeholder.com/210x295'}" alt="${episode.name}">
+      <div class="episode-content">
+        <h2>${episode.name}</h2>
+        <p>${episode.summary || 'No description available.'}</p>
+        <a href="https://www.tvmaze.com/episodes/${episode.id}" target="_blank">View on TV Maze</a>
+      </div>
     </article>`;
 }
 
+function renderShows(showsToDisplay) {
+  const showsGrid = document.getElementById('shows-grid');
```

Comment:
What are the risks of innerHTML? Why does the prep content not use this strategy?